src/components/organisms/AdminVenueView/components/AdminShowcase/AdminShowcase.tsx

```diff
@@ -5,18 +5,12 @@ import "./AdminShowcase.scss";
 
 export interface AdminShowcaseProps {
   className?: string;
-  variant?: "internal-scroll" | "no-scroll";
 }
 
 export const AdminShowcase: React.FC<AdminShowcaseProps> = ({
   className,
-  variant = "internal-scroll",
   children,
 }) => {
-  const containerClasses = classNames(
-    "AdminShowcase",
-    className,
-    `AdminShowcase--${variant}`
-  );
+  const containerClasses = classNames("AdminShowcase", className);
   return <div className={containerClasses}>{children}</div>;
 };
```

Comment:
This is all that is needed. With this (and removing the unused `className` from where it is used) I think this PR could be merged:

```tsx
export const AdminShowcase: React.FC = ({ children }) => (
  <div className="AdminShowcase">{children}</div>
);
```

Reply:
Also, a minor change would be to simply remove `&__main` in `RunTabView.scss` . Page works OK without it now.

Reply:
removed and moved some styles in the `RunTabView.scss` to have the same view: https://github.com/sparkletown/sparkle/pull/2697/commits/78a039d05022790d3f2b8a12ef645a38ca956836